hive-connector/src/main/java/org/apache/iotdb/hive/TsFileDeserializer.java

```diff
@@ -90,55 +91,55 @@ public Object deserialize(List<String> columnNames, List<TypeInfo> columnTypes,
             row.add(((BooleanWritable)data).get());
           }
           else {
-            throw new TsFileSerDeException("Unexpected data type: " + data.getClass().getName() + " for Date TypeInfo: " + type);
+            throw new TsFileSerDeException(UNEXPECTED_DATA_TYPE + data.getClass().getName() + FOR_DATE_TYPEINFO + type);
```

Comment:
String.format("Unexpected data type: %s for Date TypeInfo: %s", data.getClass().getName(), type)